components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/mapping/EmailLinkConfig.java

```diff
@@ -24,6 +24,9 @@
 public class EmailLinkConfig {
```

Comment:
EmailNotificationConfig or EmailNotificationConfig would be better name for this class IMO :wink: